README.md

```diff
@@ -1,9 +1,10 @@
 # Robot Management System
 
 ## Description
+

```

Comment:
This adheres to markdownlint rule [MD022](https://github.com/DavidAnson/markdownlint/blob/main/doc/md022.md).